<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lamport Auth Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topRow">
      <item>
       <widget class="QPushButton" name="btnConnect">
        <property name="text"><string>Connect</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="text"><string>Start</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="text"><string>Stop</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDisconnect">
        <property name="text"><string>Disconnect</string></property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblStatus">
        <property name="text"><string>Status: Idle</string></property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblRound">
        <property name="text"><string>Round: 0 / 0</string></property>
       </widget>
      </item>
     </layout>
    </item>

    <item>
     <widget class="QTextEdit" name="txtLog">
      <property name="readOnly"><bool>true</bool></property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
